spec/unit/stats/cleaner_spec.rb

```diff
@@ -7,7 +8,7 @@ module Stats
         include TestHelpers::Sequences
 
         let(:storage) { Backend::Storage.instance }
-        let(:non_proxied_instances) { storage.send(:non_proxied_instances) }
+        let(:non_proxied_instances) { [storage] }
```

Comment:
Probably we can mass-rename this variable, something like `storage_instances`?

Or even we could:
- drop this variable
- use `storage` instead
- change the methods of the Cleaner class to wrap into an array if the first param is not an array.

But probably would be better to do it in another PR.